### ISSUE TITLE
For a Serverless API there is a wskdeploy manifest available for a user to easily instantiate the PRODUCT

### ISSUE THREAD

**Issue by krook** [IBM/ogs-serverless-apis#5]

**Comment by krook** [comment]
@mrutkows  Please assign. Assuming @daisy-ycguo or @lzbj.

**Comment by mrutkows** [comment]
@krook are we going to use the Aha! generated issues?

**Comment by krook** [comment]
Need to also consider "Deploy to Bluemix" type of button for driving `wskdeploy` and doing continuous testing to ensure the app doesn't break with upstream changes.

**Comment by lzbj** [comment]
Yes, https://github.com/openwhisk/openwhisk-wskdeploy/issues/156

**Comment by mrutkows** [comment]
@lzbj @krook Please do not link back to the IBM org. issues form the OpenWhisk .org issues.  We may have to delete https://github.com/openwhisk/openwhisk-wskdeploy/issues/156 as u cannot remove the link (even as an admin.).

**Comment by mrutkows** [comment]
Please use instead https://github.com/openwhisk/openwhisk-wskdeploy/issues/159 which does not have the IBM issue reference.

**Comment by krook** [comment]
Related to #47
